chapter_appendix_math/statistics_vn.md

```diff
@@ -129,13 +129,14 @@ In this section, we introduce three common methods to evaluate and compare estim
 ### Mean Squared Error
 -->
 
-### *dịch tiêu đề phía trên*
+### Sai số toàn phương trung bình
```

Comment:
```suggestion
### Sai số trung bình bình phương
```

"sai số trung bình (của) bình phương" != "sai số bình phương (của) trung bình".

Đến đây anh lại thấy dịch là "sai số trunh bình toàn phương" nghe hợp lý hơn, đỡ bị lặp từ "bình".